### DISCUSSION TITLE
Stocks.php - Default deletion warning

### DISCUSSION THREAD

**Discussion by andrewcouling** [timschofield/webERP#277] [Development Discussion & Specification]
Since the message boxes were re-designed, I feel that the deletion warning message which is played on Stocks.php when the page loads in unnecessarily alarming. I would suggest that this warning only needs to be displayed once the 'Delete This Item'  button is clicked. Currently, when the 'Delete This Item'  is clicked, a dialogue box is displayed, seeking confirmation with the question 'Are you sure?'. Should we remove the default warning message box, are shift some of the wording into the dialogue/confirmation box?

![image](https://github.com/user-attachments/assets/b1f4024a-3b6b-4956-a8ea-6a53bf16ee1f)

![image-redacted_dot_app](https://github.com/user-attachments/assets/ae9c05a3-97dd-49c4-8335-6aab531ee070)

**Comment by timschofield** [comment]
Agreed. I think it's been in webERP from day 1 and it was always a bit
strange that it was only on Stocks.php.

thanks
Tim

On Thu, 5 Dec 2024 at 23:33, Andrew Couling ***@***.***>
wrote:

> Since the message boxes were re-designed, I feel that the deletion warning
> message which is played on Stocks.php when the page loads in unnecessarily
> alarming. I would suggest that this warning only needs to be displayed once
> the 'Delete This Item' button is clicked. Currently, when the 'Delete This
> Item' is clicked, a dialogue box is displayed, seeking confirmation with
> the question 'Are you sure?'. Should we remove the default warning message
> box, are shift some of the wording into the dialogue/confirmation box?
>
> image.png (view on web)
> <https://github.com/user-attachments/assets/b1f4024a-3b6b-4956-a8ea-6a53bf16ee1f>
>
> image-redacted_dot_app.png (view on web)
> <https://github.com/user-attachments/assets/ae9c05a3-97dd-49c4-8335-6aab531ee070>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/277>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LFYP7CSYCUHVXA5JOL2EDPGLAVCNFSM6AAAAABTDRW2GWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGYZDAMJRGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by andrewcouling** [reply]
I'll submit a PR for this.

**Comment by pakricard** [comment]
Commited and closed
